src/api/app/helpers/webui/buildstatus_helper.rb

```diff
@@ -0,0 +1,63 @@
+module Webui::BuildstatusHelper
+  # rubocop:disable Style/AlignHash
+  PACKAGE_STATUS_LIST = {
+      'succeeded'     => 'Package has built successfully and can be used to build further packages.',
+      'failed'        => 'The package does not build successfully. No packages have been created. Packages ' \
+                       'that depend on this package will be built using any previously created packages, if they exist.',
+      'unresolvable'  => 'The build can not begin, because required packages are either missing or not explicitly defined.',
+      'broken'        => 'The sources either contain no build description (e.g. specfile), automatic source processing failed or a ' \
+                       'merge conflict does exist.',
+      'blocked'       => 'This package waits for other packages to be built. These can be in the same or other projects.',
+      'scheduled'     => 'A package has been marked for building, but the build has not started yet.',
+      'dispatching'   => 'A package is being copied to a build host. This is an intermediate state before building.',
+      'building'      => 'The package is currently being built.',
+      'signing'       => 'The package has been built successfully and is assigned to get signed.',
+      'finished'      => 'The package has been built and signed, but has not yet been picked up by the scheduler. This is an ' \
+                       'intermediate state prior to \'succeeded\' or \'failed\'.',
```

Comment:
Use double quotes when you have single quotes in your string. Or even better default to double quotes:-)